File: BAEKJOON/Math/python/1193.py
# 1193: 분수 찾기/브론즈 1

x = int(input())

right = 1
term = 2
cnt = 1

while x > right:
    right += term
    term += 1
    cnt += 1

term = right - x
if cnt % 2 == 0:
    print(f'{cnt - term}/{1 + term}')
else:
    print(f'{1 + term}/{cnt - term}')

File: BAEKJOON/Math/javascript/1193.js
/*
  1193: 분수 찾기/브론즈 1
*/
const fs = require('fs');
const localPath = __dirname + '/input.txt';  // 로컬 실행용 경로
const baekPath = '/dev/stdin';  // 백준 제출용 경로
let input = +fs.readFileSync(localPath).toString();

let right = 1;
let cnt = 1;
let term = 2;

while (input > right) {
  right += term;
  cnt += 1;
  term += 1;
}


term = right - input;
if (cnt % 2 === 0) {
  console.log(`${cnt - term}/${1 + term}`);
} else {
  console.log(`${1 + term}/${cnt - term}`);
}